HLTrigger/special/plugins/HLTPixelTrackFilter.cc

```diff
@@ -50,6 +52,7 @@ HLTPixelTrackFilter::~HLTPixelTrackFilter() = default;
 
 void HLTPixelTrackFilter::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
   edm::ParameterSetDescription desc;
+  makeHLTFilterDescription(desc);
```

Comment:
If I read correctly

https://github.com/cms-sw/cmssw/blob/94ac0f5e7d34867b96581677d6e5fb94df40f40c/HLTrigger/HLTcore/src/HLTFilter.cc#L25

this will enable saving the tracks by default (which is what in my opinion is dangerous)

Reply:
Some historical background: many(!) years ago TSG moved the default from FALSE to TRUE (too many trigger proponents forgetting to flip to true - and complaining - , while filesize increase was miniscule, as only 4-vector info of objects is packed up).

Reply:
OK, I see - thanks. Still I think it would be good to have a quantitative measurement.